doc/content/modules/user-manual/pages/release-notes/2025.4.0.adoc

```diff
@@ -310,6 +310,10 @@ action def ActionDef1 {
 }
 ```
 
+- Remove the imported `EAnnotation` from published libraries
```

Comment:
Could you update this sentence to make it less technical please?
Just write what has changed for the end users.